### ISSUE TITLE
env doesn't like unsetting variables

### ISSUE THREAD

**Issue by Somasis** [landley/toybox#7]
Test case:
```
env BLAH=/dev/null BLUH="" env | grep -e BLAH -e BLUH
```

On a system using GNU coreutils, this gives:
```
somasis/discord:~/ % env BLAH=/dev/null BLUH="" env | grep -e BLAH -e BLUH
BLAH=/dev/null
BLUH=
```

On toybox, the env invocation gives this:
```
somasis/cicero:~/ % env BLAH=/dev/null BLUH=""
env: exec BLUH: No such file or directory
```

**Comment by unforgettableid** [comment]
Hi,

Why did you close this bug report?

**Comment by somasis** [comment]
It was fixed in c82be78.
